extensions/ruff/src/ruff.rs

```diff
@@ -89,7 +89,10 @@ impl RuffExtension {
             .ok_or_else(|| format!("no asset found matching {:?}", asset_name))?;
 
         let version_dir = format!("ruff-{}", release.version);
-        let binary_path = format!("{version_dir}/{asset_stem}/ruff");
+        let binary_path = match platform {
```

Comment:
I think we usually use #[cfg(windows)] for these, rather than a runtime check. See:
https://github.com/zed-industries/zed/blob/2baa704af7e6150e99da63d19a9fa42223e8acce/crates/node_runtime/src/node_runtime.rs#L28-L32

Reply:
Hi, I'm not very familiar with WASM. If we change it to a configuration-based approach, will there be any issues compiling the extension to WASM? I've noticed that other extensions also use runtime checks.

Reply:
yeah, I think for extensions it's fine to use the runtime-checks.